rules/aws_cloudtrail_rules/aws_waf_disassociation.yml

```
@@ -0,0 +1,123 @@
+AnalysisType: rule
+Description: Detection to alert when a WAF disassociates from a source.
+DisplayName: AWS WAF Disassociation
+Enabled: true
+Filename: aws_waf_disassociation.py
+Reference: https://attack.mitre.org/techniques/T1078/
+Severity: Critical
+Tags:
+    - 'MITRE ATT&CK: TA0004:T1078'
```

You'll want this to match [this format](https://github.com/panther-labs/panther-analysis/blob/cd220c87982011d4ad156c7daecd2857c358d154/rules/osquery_rules/osquery_linux_logins_non_office.yml#L13-L15) for this to show in the MITRE Matrix

```suggestion
Reports:
  MITRE ATT&CK:
    - TA0004:T1078
```